src/main/java/core/basesyntax/RandoM.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandoM extends Enum {
+    public Color getRandomColor() {
+        Random random = new Random();
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
+    }
```

Comment:
Creating a new instance of 'Random' every time the method is called is inefficient. Consider creating a single 'Random' instance as a class-level field and reusing it in both 'getRandomColor' and 'getRandomNumber' methods.